### DISCUSSION TITLE
Mix content error after login success.

### DISCUSSION THREAD

**Discussion by nnxxnx** [orchidsoftware/platform#1671] [Q&A]
We are using HTTPS on orchid admin, after login success, we get this error, can not redirect successfully to admin panel:

Mixed Content: The page at 'https://XXX/admin/login' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://XXX/admin'. This request has been blocked; the content must be served over HTTPS.

But after refresh the page, it works right way, and other pages are all work well.
Any idea about this problem?
I have set URL::forceScheme('https') in AppServiceProvider, and the browser shows the request is sent from here:
![image](https://user-images.githubusercontent.com/4382476/114993172-22d88980-9ece-11eb-937d-1df29e452e74.png)

**Comment by tabuna** [comment]
Hi, this is a problem that does not depend on this package. 
Most likely, the web server does not set the `$_SERVER ['HTTPS']` global variable to `on`.
